pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/ProtoWriterTools.java

```diff
@@ -87,15 +88,16 @@ private static RuntimeException unsupported() {
      * @param out The data output to write to
      * @param field the descriptor for the field we are writing
      * @param value the int value to write
+     * @param skipDefault default value results in no-op for non-oneOf
      */
-    public static void writeInteger(WritableSequentialData out, FieldDefinition field, int value) {
+    public static void writeInteger(WritableSequentialData out, FieldDefinition field, int value, boolean skipDefault) {
```

Comment:
It makes sense to create overloaded method 
```
public static void writeInteger(WritableSequentialData out, FieldDefinition field, int value) {
   writeInteger(out, field, value, true);
}
```
This will reduce your changeset.

Reply:
All right. This should also help reduce any conflicts with code in hedera-services, I suppose. So I introduced the overloaded helper methods as you suggest.